cmd/cri-containerd/options/options.go

```diff
@@ -158,6 +161,8 @@ func (c *CRIContainerdOptions) AddFlags(fs *pflag.FlagSet) {
 		defaults.ProfilingPort, "Profiling port for web interface host:port/debug/pprof/.")
 	fs.StringVar(&c.ProfilingAddress, "profiling-addr",
 		defaults.ProfilingAddress, "Profiling address for web interface host:port/debug/pprof/.")
+	fs.BoolVar(&c.SkipImageFSUUID, "skip-imagefs-uuid",
+		defaults.SkipImageFSUUID, "Skip retrieving imagefs uuid. When turned on, kubelet can't get imagefs capacity and imagefs disk eviction can't work.")
```

Comment:
Skip retrieval of imagefs uuid for new CRIContainerdService instances. When turned on, the kubelet will not be able to get imagefs capacity or request imagefs disk eviction.

Reply:
`CRIContainerdService` is code name, I don't think user cares about that in the flag discription. :)

Reply:
Done.